src/dataflow-types/src/types.rs

```diff
@@ -381,6 +381,7 @@ impl DataEncoding {
             DataEncoding::Protobuf(ProtobufEncoding {
                 descriptors,
                 message_name,
+                ..
```

Comment:
> generally we prefer to explicitly list all fields in match cases so that changes to items force consideration of everywhere that could be affected:

Hrm, this definitely isn't a rule I've heard of us enforcing, besides some very special code in SQL normalization.